### PR TITLE
#2764 Add permissions to Lnb extension configuration

### PR DESCRIPTION
**Pull request by minhyun2**
### Description
Add permissions to Lnb extension configuration

**Related Issue** : #2764 

### How Has This Been Tested?

1. Add the first and second depth menus, referring to the bottom of the guide in the link below.
[https://github.com/metatron-app/metatron-discovery/blob/f%232764-add_permissions_to_lnb/discovery-server/src/main/asciidoc/application-config-guide.adoc#app-properties](https://github.com/metatron-app/metatron-discovery/blob/f%232764-add_permissions_to_lnb/discovery-server/src/main/asciidoc/application-config-guide.adoc#app-properties)
2. If you do not add a value for permissions, all users with permissions in the upper depth menu can access.
3. If you add a value for permissions, the extension menu will be displayed for the user corresponding to the permission you added.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

**Comment by minhyun2** [comment]
run build
deploy to 5

**Comment by minhyun2** [comment]
deploy to 5

**Comment by kyungtaak** [comment]
@minhyun2 you have to add comment application configuration guide page :) (https://github.com/metatron-app/metatron-discovery/blob/master/discovery-server/src/main/asciidoc/application-config-guide.adoc )

**Comment by minhyun2** [comment]
@kyungtaak I add comment application configuration guide page

File: discovery-server/src/main/asciidoc/application-config-guide.adoc [hunk 1]
```diff
@@ -146,5 +146,10 @@ polaris:
       timeout: 20       # Timeout for rule edit sessions
     etl:
       timeout: 36000    # Timeout for generating snapshots
+  extensions:
+      lnb:
```
Comment (kyungtaak) [inline discovery-server/src/main/asciidoc/application-config-guide.adoc]: @minhyun2 you should be mentioned that the menu can be expanded in the lnb area.

File: discovery-server/src/main/asciidoc/application-config-guide.adoc [hunk 2]
```diff
@@ -146,5 +146,10 @@ polaris:
...
+        - name: Engine Monitoring   # for Engine(Druid) Monitoring
```
Comment (kyungtaak) [inline discovery-server/src/main/asciidoc/application-config-guide.adoc]: @minhyun2 I think there should be a description of parent

Comment (minhyun2) [reply discovery-server/src/main/asciidoc/application-config-guide.adoc]: @kyungtaak 
The Engine Monitoring menu is in Beta. Therefore, it was added to the configuration regardless of parent.

**Comment by minhyun2** [comment]
deploy to 5

**Comment by minhyun2** [comment]
deploy to 5

**Comment by kyungtaak** [comment]
deploy to 5

**Comment by minhyun2** [comment]
deploy to 5

File: discovery-server/src/main/asciidoc/application-config-guide.adoc (hunk 1 above)
Comment (minhyun2) [reply discovery-server/src/main/asciidoc/application-config-guide.adoc]: @kyungtaak 
I added a comment as below.
# Specify the menu to add to the LNB.

**Pull request merged**